src/theme/typography.stories.mdx

```diff
@@ -12,15 +12,22 @@ leo tincidunt vehicula non a dolor. Suspendisse placerat turpis nisl, quis gravi
 # Typography
 
 Design System have defined two basic font families. As base font family we are using sans-serif
-`ProximaNova` font, for monospaced text we are using `SpaceMono` font.
+`Inter` font, for monospaced text we are using `SpaceMono` font.
 
 ## Font licensing
 
-### ProximaNova
+### Inter
 
-ProximaNova is licensed font that cannot be bundled and redistributed along with this package. If you
-want to use this font you have to obtain your own license for this font. You can use for example
-[Adobe Fonts](https://fonts.adobe.com/fonts/proxima-nova) service.
+Inter is licensed under the [Open Font License](https://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&id=OFL).
+It is not bundled with the package due to its size, therefore it needs to be included in your html imports.
+
+Example:
+
+```html
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600&display=swap" rel="stylesheet">
```

Comment:
Linking to Google fonts can be way for some but it's not applicable to use due to company restrictions. Let's also add self-hosting options like using NPM package (https://www.npmjs.com/package/@fontsource/inter) or direct download of font files from https://rsms.me/inter/download/